api/models/userModel.js

```diff
@@ -1,40 +1,71 @@
+/**
+ * userModel.js - Module for handling user-related operations.
+ * This module provides functions for creating users, signing in users, and retrieving user information.
+ */
+
+// Import required modules
 const db = require('../db');
 const crypto = require('crypto');
 const { QueryFile } = require('pg-promise');
+const path = require('path');
 
+// Define queries for user operations
 const userQueries = {
-    createUser: new QueryFile('./sql/userSQL/create.sql'),
-    createUserData: new QueryFile('./sql/userDataSQL/create.sql'),
-    signInUser: new QueryFile('./sql/userSQL/signIn.sql'),
-    getUserInfo: new QueryFile('./sql/userDataSQL/get.sql')
+    createUser: new QueryFile(path.join(__dirname, './sql/userSQL/create.sql')),
+    createUserData: new QueryFile(path.join(__dirname, './sql/userDataSQL/create.sql')),
+    signInUser: new QueryFile(path.join(__dirname, './sql/userSQL/signIn.sql')),
+    getUserInfo: new QueryFile(path.join(__dirname, './sql/userDataSQL/get.sql'))
 };
 
+/**
+ * Retrieves user information based on a key.
+ * @param {string} key - The key used to retrieve user information.
+ * @returns {Promise<Object>} The user information.
+ */
 async function getUserInfo(key) {
```

Comment:
the key should be specified as the session key, it clear up ambiguity. Sorry, thats on me.